File: Python/백준/Silver/2751. 수 정렬하기 2/수 정렬하기 2.py
import sys
input = sys.stdin.readline
def main():
    N = int(input().strip())
    arr = []
    for _ in range(N):
        arr.append(int(input().strip()))
    # 정렬 (O(N log N))
    arr.sort()
    # 출력 — join + write 방식이 빠르다
    sb = []
    for x in arr:
        sb.append(str(x))
    sys.stdout.write("\n".join(sb))

if __name__ == "__main__":
    main()

File: Java/백준/Silver/2751. 수 정렬하기 2/수 정렬하기 2.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.BufferedWriter;
import java.io.OutputStreamWriter;
import java.io.IOException;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int N = Integer.parseInt(br.readLine().trim());
        int[] arr = new int[N];
        for (int i = 0; i < N; i++) {
            arr[i] = Integer.parseInt(br.readLine().trim());
        }

        Arrays.sort(arr);

        for (int x : arr) {
            bw.write(Integer.toString(x));
            bw.newLine();
        }
        bw.flush();
        bw.close();
        br.close();
    }
}